playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -98,6 +98,17 @@
   tasks:
   - set_fact:
       openshift_rpm: "{{ hostvars[groups['nodes'][0]]['yum'].results | selectattr('name', 'match', '^(origin|atomic-openshift)$') | first }}"
+  - name: create a file with image name
+    copy:
+      content: |
+        ---
+        image_name: {{ openshift_azure_output_image_name }}
+        base_image: {{ (input_image.stdout | from_json).name }}
+        subscription: {{ lookup('env', 'AZURE_SUBSCRIPTION_ID') }}
+      dest: /etc/origin/image.yml
+      owner: root
+      group: root
+      mode: '0644'
```

Comment:
0444 or 0440?  No one needs to write to it after it is put down.  Not that it would stop a root user, I guess :smile: